packages/trackerless-network/src/logic/StreamPartEntryPointDiscovery.ts

```diff
@@ -116,24 +111,9 @@ export class StreamPartEntryPointDiscovery {
     private async queryEntrypoints(key: Uint8Array): Promise<PeerDescriptor[]> {
         logger.trace(`Finding data from dht node ${this.config.ownPeerDescriptor.nodeName}`)
         try {
-            const results = await this.config.getEntryPointData(key)
-            if (results.dataEntries) {
-                return parseEntryPointData(results.dataEntries)
-            } else {
-                return []
-            }
-        } catch (err) {
-            return []
-        }
-    }
-
-    // TODO remove this method in NET-1122
-    private async queryEntryPointsViaNode(key: Uint8Array, node: PeerDescriptor): Promise<PeerDescriptor[]> {
-        logger.trace(`Finding data via node ${this.config.ownPeerDescriptor.nodeName}`)
-        try {
-            const results = await this.config.getEntryPointDataViaNode(key, node)
-            if (results) {
-                return parseEntryPointData(results)
+            const result = await this.config.getEntryPointData(key)
+            if (result) {
```

Comment:
The `result` can't be undefined by the type definition. Could maybe remove if statement?